frontend/src/components/WarningBanner/Warning.tsx

```diff
@@ -11,7 +11,7 @@ export const CrossfeedWarning: React.FC = (props) => {
     borderWidth: 2,
     padding: theme.spacing(1),
     backgroundColor: 'inherit',
-    position: 'absolute',
+    // position: 'absolute',
```

Comment:
Delete this instead of commenting it out.

Reply:
i thought i did..lol